macros/anaUltraThreshold.py

```diff
@@ -161,15 +161,22 @@
 vt1 = []
 for i in range(0,24):
 #    vt1.append([])
-    for j in range(0, 100):
+    for j in range(0, 250):
```

Comment:
What is the meaning of 250 here (and 251 in line 123)?
Is it related to something hardcoded elsewhere?